src/NodeApi/Interop/NodeStream.cs

```diff
@@ -18,7 +18,7 @@ public partial class NodeStream : Stream
 
     public static explicit operator NodeStream(JSValue value) => new(value);
     public static implicit operator JSValue(NodeStream stream)
-        => stream._valueReference.GetValue() ?? default;
```

Comment:
It is another place where we start to throw.

Reply:
It is always a strong reference so it could never have returned null before.